Solutions/Azure Active Directory/Analytic Rules/MaliciousOAuthApp_O365AttackToolkit.yaml

```diff
@@ -71,5 +74,9 @@ entityMappings:
     fieldMappings:
       - identifier: Address
         columnName: IPCustomEntity
-version: 1.0.0
-kind: Scheduled
\ No newline at end of file
+  - entityType: CloudApplication
+    fieldMappings:
+      - identifier: Name
+        columnName: AppDisplayName
+version: 1.0.1
```

Comment:
Everything else looks good to me. Thank you for your patience. I will confirm with the author before the approval.
A small change, the changes made are modification in the logic. Thus version should be 1.1.0 now.

Reply:
<img width="300" alt="Screenshot 2023-03-08 at 6 41 55 AM" src="https://user-images.githubusercontent.com/20403061/223704553-9b73b152-be1c-4b52-a9fd-f6742736c0e6.png">